packages/router/src/router.ts

```diff
@@ -52,10 +55,19 @@ export const makeRouter = async () => {
     context.wallet = context.config.mnemonic
       ? Wallet.fromMnemonic(context.config.mnemonic)
       : new Web3Signer(context.config.web3SignerUrl!);
+    context.signerAddress = await context.wallet.getAddress();
+    context.isRouterContract = context.config.routerContractAddress ? true : false;
```

Comment:
`!!context.config.routerContractAddress` ?

Reply:
it throws `Redundant double negation` when I tried using it.